kubernetes/src/main/resources/kubernetes/catalog.bom

```diff
@@ -0,0 +1,5 @@
+brooklyn.catalog:
+  items:
+    - classpath://docker/catalog.bom
```

Comment:
How does this work outside of Brooklyn classic? This is in a different module, so a different OSGi bundle. Would we not need the bundleId in the URL as well?